release.csx

```diff
@@ -0,0 +1,84 @@
+#r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
+#r ".\packages\system.runtime.interopservices.runtimeinformation\4.0.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
+
+using Octokit;
+
+const string repoOwner = "FakeItEasy";
+const string repoName = "FakeItEasy";
+const string remote = "upstream";
+
+var version = Args.FirstOrDefault();
+if (version == null)
+{
+    throw new Exception("No version supplied");
+}
+
+var releaseBranchName = $"release/{version}";
+
+var token = ReadTokenFromNetrc();
+
+var gitHubClient = new GitHubClient(new ProductHeaderValue("FakeItEasy-build-scripts"));
+gitHubClient.Credentials = new Credentials(token);
+Cmd("git", $"checkout -b {releaseBranchName}");
+Cmd("git", $"push {remote} {releaseBranchName}");
+
+Console.WriteLine($"Creating pull request to merge {releaseBranchName} into master...");
+var pr = await gitHubClient.PullRequest.Create(
+    repoOwner,
+    repoName,
+    new NewPullRequest($"Release {version}", releaseBranchName, "master"));
+Console.WriteLine($"Created pull request {pr.Number}: '{pr.Title}'");
+
+
+public static string ReadTokenFromNetrc()
+{

```

Comment:
I think this is "good enough", but not particularly robust.

1. assumes the .netrc file is called {{.netrc}}. Some may call {{_netrc}}.
2. assumes a {{HOME}} environment variable, in which .netrc lives. This is standard, and would've been needed for our ruby implementation
3. only looks for api.github.com. Technically we should use the "default" entry as well, but that just seems silly

Reply:
1. Fine by me. If we want to add support for `_netrc` later, it won't be hard anyway.
2. `HOME` isn't a standard environment variable on Windows (I realized this recently at work, when I used it in a script and it didn't work on my colleagues' machines). The equivalent is `USERPROFILE`.
3. Fine by me

Reply:
Yeah, I know `HOME` is nonstandard, but the .netrc-using tools all use it. I was going to suggest falling back to `USERPROFILE` if you were concerned. But I'm content to take your other suggestion and just look for a file in the repo.